src/content/docs/new-relic-control/agent-control/troubleshooting.mdx

```diff
@@ -0,0 +1,116 @@
+---
```

Comment:
I added in the preview docs this section that I think it makes sense to keep https://github.com/newrelic/beta-docs-site/pull/478/files